lib/parser/unparse.cc

```diff
@@ -1757,9 +1757,11 @@ class UnparseVisitor {
   }
   // OpenMP Clauses & Directives
   void Unparse(const OmpObject &x) {
-    bool isCommon{std::get<OmpObject::Kind>(x.t) == OmpObject::Kind::Common};
-    const char *slash{isCommon ? "/" : ""};
-    Put(slash), Walk(std::get<Designator>(x.t)), Put(slash);
+    if (const auto *designator{std::get_if<Designator>(&x.u)}) {
+      Walk(*designator);
+    } else if (const auto *name{std::get_if<Name>(&x.u)}) {
+      Put("/"), Walk(*name), Put("/");
+    }
```

Comment:
`std::visit` is better here. This makes it looks like there could be other alternatives that are begin ignored. And if a third alternative were ever added, the `std::visit` form would get a compilation error that reminds you to add something here.

Reply:
Okay I'll change that. Thanks

Reply:
Updated.